Source/Alamofire.swift

```diff
@@ -25,7 +25,7 @@
 import Foundation
 
 /// Global namespace containing API for the `default` `SessionManager` instance.
-public enum Alamofire {
+public enum AF {
```

Comment:
👏🏻👏🏻👏🏻

Reply:
AF?

Reply:
Avoid confusion with the Alamofire namespace. Now it's no longer `Alamofire.Alamofire.request...`